syntaxes/gleam.tmLanguage.json

```
@@ -15,15 +21,83 @@
       "include": "#constant"
     },
     {
-      "include": "#entity"
+      "include": "#punctuation"
     }
   ],
   "repository": {
+    "support": {
+      "patterns": [
+        {
+          "name": "support.function.gleam",
+          "match": "\\b(todo)\\b"
+        }
+      ]
+    },
+    "punctuation": {
+      "patterns": [
+        {
+          "name": "punctuation.accessor.gleam",
+          "match": "(\\.)"
+        },
+        {
+          "name": "punctuation.separator.comma.gleam",
+          "match": "(,)"
+        }
+      ]
+    },
     "keywords": {
       "patterns": [
         {
-          "name": "keyword.control.gleam",
-          "match": "\\b(use|case|if|external|fn|import|let|assert|try|pub|type|opaque|const|todo)\\b"
+          "name": "keyword.control.case.gleam",
+          "match": "\\b(case)\\b"
+        },
+        {
+          "name": "keyword.control.conditional.gleam",
+          "match": "\\b(if)\\b"
+        },
+        {
+          "name": "keyword.control.trycatch.gleam",
+          "match": "\\b(try|assert)\\b"
```

Since `try` is no longer a language keyword: 
```suggestion
          "match": "\\b(assert)\\b"
```

---

Can we call it `assertion` or something rather than `trycatch`?